projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-method.component.ts

```diff
@@ -96,17 +96,30 @@ export class PaymentMethodComponent implements OnInit, OnDestroy {
           textDefaultPaymentMethod,
           textSelected,
         ]) => {
-          let ccImage: string;
+          let ccIcon: string;
```

Comment:
Could you please move this logic to another function. So maybe like `private getCardIcon(code: string)` that returns the code and call it from here.